parametric/protos/apm_test_client.proto

```diff
@@ -10,6 +10,18 @@ service APMClient {
   rpc InjectHeaders(InjectHeadersArgs) returns (InjectHeadersReturn) {}
   rpc FlushSpans(FlushSpansArgs) returns (FlushSpansReturn) {}
   rpc FlushTraceStats(FlushTraceStatsArgs) returns (FlushTraceStatsReturn) {}
+
+  rpc OtelStartSpan(OtelStartSpanArgs) returns (OtelStartSpanReturn) {}
+  rpc OtelEndSpan(OtelEndSpanArgs) returns (OtelEndSpanReturn) {}
+  rpc OtelIsRecording(OtelIsRecordingArgs) returns (OtelIsRecordingReturn) {}
+  rpc OtelSpanContext(OtelSpanContextArgs) returns (OtelSpanContextReturn) {}
+  rpc OtelSetStatus(OtelSetStatusArgs) returns (OtelSetStatusReturn) {}
+  rpc OtelSetName(OtelSetNameArgs) returns (OtelSetNameReturn) {}
+  rpc OtelSetAttributes(OtelSetAttributesArgs) returns (OtelSetAttributesReturn) {}
+  rpc OtelFlushSpans(OtelFlushSpansArgs) returns (OtelFlushSpansReturn) {}
+  rpc OtelFlushTraceStats(OtelFlushTraceStatsArgs) returns (OtelFlushTraceStatsReturn) {}
```

Comment:
It feels like the original service is duplicate. Why don't create another service?

Reply:
It started as a separate service but was advised against to reduce area of code support. Alternatively, I could create an almost  duplicate of APMCLientStub to split the two and pass in an analogous test_otel_library